dev-php/scrypt/scrypt-1.4.2.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PHP_EXT_ECONF_ARGS="--enable-scrypt"
+USE_PHP="php7-2 php7-3 php7-4"
```

Comment:
Could you update this to reflect today? Ie, php-7.2 is masked and getting removed soon, and there's php-8.0 available.

Reply:
Updated.
1. PHP 7.2 removed
2. PHP 8.0 added with patch from upstream's master.